tests/services/test_statuses_service.py

```diff
@@ -0,0 +1,38 @@
+from collections.abc import Generator, Sequence
+from typing import Any
+from unittest.mock import patch
+
+import pytest
+from tad.models import Status
+from tad.repositories.statuses import StatusesRepository
+from tad.services.statuses import StatusesService
+
+
+class MockStatusesRepository:
+    def __init__(self):
+        pass
+
+    def find_by_id(self, status_id: int) -> Status:
+        return Status(id=status_id, name="Test status", sort_order=1)
+
+    def find_all(self) -> Sequence[Status]:
+        return [self.find_by_id(1)]
+
+
+@pytest.fixture(scope="module")
+def mock_statuses_repository() -> Generator[MockStatusesRepository, Any, Any]:
+    with patch("tad.services.statuses.StatusesRepository"):
+        mock_statuses_repository = MockStatusesRepository()
```

Comment:
You could consider using MagicMock Instead of making your own

Reply:
I think I have looked into that. I could not get things working the way I wanted. I'd prefer to create a mock based on the original class, and implement functions or responses as needed. If MagicMock does that, will refactor a next iteration.